SparkPost/2web2ui
# FAD-5629 Suppressions - Upload List

_Refer to [FAD-5629](https://jira.int.messagesystems.com/browse/FAD-5629) for AC._  Note: entering a single entry is not included and will be addressed in a later ticket.

**Before**

<img width="1385" alt="screen shot 2018-01-25 at 3 58 45 pm" src="https://user-images.githubusercontent.com/1335605/35411950-a9aaba9a-01e8-11e8-91d3-f2cbb34837ea.png">

**After**

<img width="1378" alt="screen shot 2018-02-07 at 1 34 27 am" src="https://user-images.githubusercontent.com/1335605/35901906-1d5f2620-0ba7-11e8-8be0-4137816b9057.png">

**How to test?**
- Use these [examples.zip](https://github.com/SparkPost/2web2ui/files/1702002/Archive.zip) files to test different states.
- Quick sanity check that recipient list upload still works as expected.
